lib/api/core/models/repositories/repository.js

```diff
@@ -131,14 +130,14 @@ class Repository {
         ids));
     }
 
-    return this.indexEngine.mGet(this.collection, ids)
-      .then(response => {
-        if (!response.hits || response.hits.length === 0) {
+    return this.indexStorage.mGet(this.collection, ids)
+      .then(({ items }) => {
+        if (! items.length === 0) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "false". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS2583)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this "===" check; it will always be false. Did you mean to use "=="? [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3403)